README.md

```diff
@@ -196,3 +196,29 @@ string is constructed from the following environment variables:
 * `POSTGRESQL_PASSWORD` - defaults to `coreapi` in the local setup
 * `POSTGRESQL_DATABASE` - defaults to `coreapi` in the local
 * `PGBOUNCER_SERVICE_HOST` - defaults to `coreapi-pgbouncer` (note that PostgreSQL/Amazon RDS is accessed through PGBouncer, thus naming)
+
+### Importing the production graph data
+
+There can be requirements where a developer will have to import the production data
+on her/his development graph database. Following process has to be followed for the
+data import:
+
+1. SRE team dumps production data on the S3 bucket `dynamodb-migration-backups-osio`
+in `osio-dev` namespace.
+2. Use the AWS Data pipeline service to import data from graph tables.
+3. Create a pipeline with the source template `Import DynamoDB backup data from S3`.
+4. Select one of the backup folders in `s3://dynamodb-migration-backups-osio/backups/prod_edgestore/`
+as `Input S3 folder`.
+5. Provide the corresponding developer dynamodb table name as `Target DynamoDB table name`.
```

Comment:
Dynamodb table should be created before this step. So two steps should talk about create dynamodb table with name and use `hk` and `rk` as index and sort key

Reply:
@miteshvp this assumes that the user already has a working dev cluster instance. The steps above are all those creating a new instance.

Reply:
Ok makes sense.